fastapi/README.rst

```diff
@@ -1006,49 +1006,63 @@ normally provided by the normal processing of the request by the fastapi app.
 to test the behavior of your route handlers when the partner is not authenticated,
 you can also inject a mock for the odoo_env etc...)
 
-With all these features, writing a test for the 'Hello world' route handler
-defined into the demo app is as simple as
+The fastapi addon provides a base class for the test cases that you can use to
+write your tests. This base class is **'odoo.fastapi.tests.common.FastAPITransactionCase'**.
+This class mainly provides the method **'_create_test_client'** that you can
+use to create a test client for your fastapi app. This method encapsulates the
+creation of the test client and the injection of the dependencies. It also
+ensures that the odoo environment is make available into the context of the
+route handlers. This method is designed to be used when you need to test your
+app or when you need to test a specific router (It's therefore easy to defines
+tests for routers in an addon that doesn't provide a fastapi endpoint).
+
+With this base class, writing a test for a route handler is as simple as:
 
 .. code-block:: python
 
-  from functools import partial
+  from odoo.fastapi.tests.common import FastAPITransactionCase
 
-  from requests import Response
+  from odoo.addons.fastapi import dependencies
+  from odoo.addons.fastapi.routers import demo_router
 
-  from odoo.tests.common import TransactionCase
+  class FastAPIDemoCase(FastAPITransactionCase):
```

Comment:
I think that we should run the test with @tagged("post_install", "-at_install") or did it change?